docs/PreviewFeatures.md

```diff
@@ -0,0 +1,23 @@
+Preview features
+=========
+
+New features are phased in to the MsQuic library in a staged manner to enable gradual roll in, when each of these features are deemed ready for wider use.
+
+These features are first made available in a library-wide **Preview mode** till they are made part of the mainstream MsQuic library functionality.
+
```

Comment:
I would add a big red disclaimer here (maybe using the github tag below) to say that preview features are not a stable API and can be modified or removed at any time without warning.
Just to make sure nobody can say they didn't know!

```
> [!IMPORTANT]  
> Crucial information necessary for users to succeed.
```